src/docfx.website.themes/default/ManagedReference.common.js

```diff
@@ -52,7 +54,13 @@ function groupChildren(model, typeChildrenItems) {
   var grouped = {};
 
   model.children.forEach(function (c) {
-    var type = c.type.toLowerCase();
+    if (c.isEii) {
+      var type = "eii";
+    } else if (c.isExtensionMethod) {
+      var type = "extensionMethod"
```

Comment:
extensionmethod is a bit different because the method's definition isn't in the class's page, namely it won't be put in Children property.